### PR TITLE
add support for strip-trailing-cr

### PR DESCRIPTION
**Pull request by bakito**
This PR add support to strip the trailing cr from charts that have been deployed wit different line endings.

See #144

### CODE REVIEW

**Comment by amorozov87** [comment]
Can somebody merge this request?

**Comment by bakito** [comment]
Can someone please have a look at this?

File: cmd/upgrade.go [hunk 1]
```diff
@@ -270,7 +272,8 @@ func (d *diffCmd) run() error {
 		}
 	}
 
-	seenAnyChanges := diff.Manifests(currentSpecs, newSpecs, d.suppressedKinds, d.showSecrets, d.outputContext, d.output, os.Stdout)
+	println("#######################################################################################################################################*******************************************************")
```
Comment (mumoshu) [inline cmd/upgrade.go:275]: Can we remove this as an irrelevant change?

**Review by mumoshu — APPROVED** [review]
Sorry for the delayed response.

Just one comment on a seemingly redundant println. It would be great if you could confirm that it does look like an unnecessary change.

I'll merge this and revert the println addition only after a few days anyway.
Thanks for your contribution @bakito!

**Comment by bakito** [comment]
Hi @mumoshu 
I remove the debug line, sorry I missed that.

**Comment by mumoshu** [comment]
@bakito Thanks again for your support! Merging.

**Pull request merged**